video/index.js

```diff
@@ -13,6 +18,12 @@ class IrnmnVideo extends HTMLElement {
         this.setupButton();
     }
 
+    attributeChangedCallback(name, oldValue, newValue) {
```

Comment:
Can you pls add docblock here explaining why we're triggering connectedCallback based on src change?